playbooks/freeipa.yml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: all
+  become: true
+  vars:
+    server_group: "server-{{ inventory_hostname }}"
+    server: "{{ groups[server_group][0] }}"
+  roles:
+    - etc_hosts
+    - katello_repositories
```

Comment:
The plan is to have a capsule installed with freeipa and also hosting the freeipa server. I'm still working on this, though.